### PR TITLE
Detiles Popup Window

### PR DESCRIPTION
**Pull request by aliveGUY**
# :rocket:Create Pop-up Window With Project Details:rocket:
## Main Changes:checkered_flag: :
:heavy_check_mark: Delete previous HTML work-card layout
:heavy_check_mark: Create scriptable object work-card
:heavy_check_mark: Create local mini database
:heavy_check_mark: Create scriptable object popup
:heavy_check_mark: Fix mobile menu bug

### CODE REVIEW

**Comment by aliveGUY** [comment]
# :warning::warning::warning:PLEASE ADVISE WORKING SOLUTION ON HOW TO SOLVE import/prefer-default-export:warning::warning::warning:

File: src/scripts/Data.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+export const Works = [
```
Comment (keroloussamy) [inline src/scripts/Data.js]: use `export default` to solve it

**Review by keroloussamy — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linter error.

You can fix it by add `export default`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: src/scripts/Data.js (hunk 1 above)
Comment (aliveGUY) [reply src/scripts/Data.js]: Screenshot from 2022-04-06 12-23-42
Screenshot from 2022-04-06 12-24-37

Comment (aliveGUY) [reply src/scripts/Data.js:1]: # Please try the solution before advice

File: index.css [hunk 2]
```diff
@@ -158,6 +169,134 @@
...
+.Popup {
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
+  color: #344563;
+  font-family: 'Inter', sans-serif;
+  height: 85vh;
+  width: 85vw;
+  background: #fff;
+  padding: 16px;
+}
```
Comment (keroloussamy) [inline index.css:192]: - [ ] You did great work until now, just one thing here is you have to make the project pic on the left and the description and the buttons on the right to match the Figma design.
To solve it you can wrap them in a container and use a display flex to make them next to each other.

Your design
image

Figma design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @aliveGUY, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — APPROVED** [review]
Hi @aliveGUY, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**